### DISCUSSION TITLE
JP2 importing template

### DISCUSSION THREAD

**Discussion by hedger48** [OpenOrienteering/mapper#2095] [Questions & Answers]
Hello, i downloaded my first orthoimages from USGS and i can open them in GIMP (although the appear black and white), when i try to add a template in OOM, nothing shows up?  Is this some silly operator error?
thanks
Kelly

**Comment by dl3sdo** [comment]
Hello Kelly,
in theory Mapper should be able to handle JPEG2000 since 0.92, according to
https://github.com/OpenOrienteering/mapper/releases?page=2
However, there are various jp2 variants, some of them are supported directly by GDAL,
some not, see:
https://gdal.org/drivers/raster/index.html

There was a previous discussion regarding using USGS images:
https://sourceforge.net/p/oorienteering/discussion/mapper/thread/d1638059/
I also remember a discussion regarding ECW files, see #1910.

Can you provide a (not to large) image?
Regards, Matthias

**Comment by hedger48** [comment]
thanks, one of the the JP2 files i couldn't get to work is here
<https://drive.google.com/file/d/1bsCroVBk3rMnuEo2HHfd0D1Fvk4auk1Z/view?usp=sharing>.
it is 65M so i am sending a link.
strangely enough, today i tried doing the same trick i did earlier this
week where i opened the JP2 in QGIS and then exported as PNG, today that
didnt work either, OOM opening the template blanks white space.  So this
time i used QGIS to export to *TIFF *and i can now get OOM to open that.

In any case, i have a work around; on one hand it may be better to do it
this way since i can crop to my property before exporting to TIFF and OOM
only has to deal with the region of interest.

thanks


On Thu, Sep 22, 2022 at 3:38 AM Matthias Kühlewein ***@***.***>
wrote:

> Hello Kelly,
> in theory Mapper should be able to handle JPEG2000 since 0.92, according to
> https://github.com/OpenOrienteering/mapper/releases?page=2
> There was a previous discussion regarding using USGS images:
> https://sourceforge.net/p/oorienteering/discussion/mapper/thread/d1638059/
> Can you provide a (not to large) image?
> Regards, Matthias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenOrienteering/mapper/discussions/2095#discussioncomment-3706827>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALHJO3U72YD62HN3I6WETELV7QEINANCNFSM6AAAAAAQRPG6KU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Kelly Atkinson
512-569-1348
skype: hedger48
